### PR TITLE
Create atreticAnatomicalEntityInLocation.yaml

### PR DESCRIPTION
**Pull request by nicolevasilevsky**

### CODE REVIEW

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "atresia of the %s"
+    vars:
+     - anatomical_entity
+     - location
+
+def:
+  text: "A developmental defect resulting in complete obliteration of the lumen of the %s."
```
Comment (matentzn) [inline src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: Definition is missing the location!

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+pattern_name: atreticAnatomicalEntityInLocation
+pattern_iri: http://purl.obolibrary.org/obo/upheno/patterns-dev/atreticAnatomicalEntityInLocation
```
Comment (matentzn) [inline src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: missing the .yaml file extension.

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+description: "An anatomical entity with atresia, in a location, such as HP_0000067 'Urethral atresia, female'."
```
Comment (matentzn) [inline src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: Definition should probably be a bit more extensive, especially including a short phrase that explains what atresia is (remember, the definition should be good enough to enable general biomedical curators, not necessarily medical experts).

Comment (matentzn) [reply src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: Also, your example - is it really `in location`? In female body?

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+name:
+  text: "%s atresia"
+  vars:
+   - anatomical_entity
+   - location
+
+annotations:
+  - annotationProperty: exact_synonym
+    text: "atresia of the %s"
```
Comment (matentzn) [inline src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: missing location %s

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym 
+
+vars:
+  anatomical_entity: "'anatomical entity'"
+  location: "'independent continuant'"
+
+name:
+  text: "%s atresia"
```
Comment (matentzn) [inline src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: Missing location %s

File: src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml (hunk 3 above)
Comment (nicolevasilevsky) [reply src/patterns/dosdp-dev/atreticAnatomicalEntityInLocation.yaml]: I created this pattern for the terms:
http://purl.obolibrary.org/obo/HP_0000067	Urethral atresia, female
http://purl.obolibrary.org/obo/HP_0000052	Urethral atresia, male

Do you think this pattern works?

OH! I just realized that I changed this pattern in HPO to use the classes 'female urethra' and 'male urethra.

I don't think we need this pattern!

**Comment by nicolevasilevsky** [comment]
See comments above:
OH! I just realized that I changed this pattern in HPO to use the classes 'female urethra' and 'male urethra.

I don't think we need this pattern!

I am going to close this PR.

**Pull request closed without merging**